gui/storage/views.py

```diff
@@ -510,13 +512,15 @@ def zvol_create(request, parent):
             zvol_name = "%s/%s" % (parent, cleaned_data.get('zvol_name'))
             zvol_compression = cleaned_data.get('zvol_compression')
             props['compression'] = str(zvol_compression)
+	    zvol_comments = cleaned_data.get('zvol_comments')
             if zvol_blocksize:
                 props['volblocksize'] = zvol_blocksize
             errno, errmsg = notifier().create_zfs_vol(
                 name=str(zvol_name),
                 size=str(zvol_size),
                 sparse=cleaned_data.get("zvol_sparse", False),
                 props=props)
+	    notifier().zfs_set_option(name = str(zvol_name),item = "volume_entity:comments",value = zvol_comments)
             if errno == 0:
```

Comment:
Not sure if the FreeNAS project uses this, but several projects I've been involved in use flake8 with some subset of the options in place as a make target to be used before submitting code.  We also use Travis to cause an error if someone submits code that flake8 disagrees with.  Just need to carefully pick which flake8 tests you want to run, some are really overly annoying.